File: week6_python/sentimental-cash/cash.py
from cs50 import get_float

while True:
    try:
        change = get_float("Change: ")
        if change > 0:
            break
    except ValueError:
        print("Please input a positive number")

coins = 0
cents = round(change * 100)

while cents >= 25:
    cents = cents - 25
    coins += 1
while cents >= 10:
    cents = cents - 10
    coins += 1
while cents >= 5:
    cents = cents - 5
    coins += 1
while cents >= 1:
    cents = cents - 1
    coins += 1

print(f"{coins}")

File: week1_c/cash/cash.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int coins = 0;
    int cash;

    do{
        cash = get_int("Change owed: ");
    }while(cash < 1);

    while (cash >= 25)
    {
        cash -= 25;
        coins++;
    }
    while (cash >= 10)
    {
        cash -= 10;
        coins++;
    }
    while (cash >= 5)
    {
        cash -= 5;
        coins++;
    }
    while (cash >=1)
    {
        cash -= 1;
        coins++;
    }

    printf("%i\n", coins);
}
